src/main/java/com/asana/models/Project.java

```diff
@@ -28,15 +41,4 @@
     public DateTime createdAt;
     @SerializedName("modified_at")
     public DateTime modifiedAt;
-
-    public User owner;
-
-    public Project() {
```

Comment:
Seriously, WTF is this? How have we never noticed these wacko constructors here? Thanks for removing them.